src/defiProvider.ts

```diff
@@ -211,31 +212,44 @@ export class DefiProvider {
     filterProtocolIds,
     filterChainIds,
     filterProtocolToken,
+    filterTokenId,
     blockNumbers,
   }: {
     filterProtocolIds?: Protocol[]
     filterChainIds?: Chain[]
     filterProtocolToken?: string
+    filterTokenId?: string
     blockNumbers?: Partial<Record<Chain, number>>
   }): Promise<PricePerShareResponse[]> {
     const runner = async (adapter: IProtocolAdapter) => {
       const blockNumber = blockNumbers?.[adapter.chainId]
 
-      const protocolTokens = filterProtocolToken
+      const protocolTokens:
+        | (Erc20Metadata & {
+            tokenId?: string
+          })[]
+        | {
+            address: string
+            tokenId?: string
+          }[] = filterProtocolToken
         ? [
             {
               address: filterProtocolToken,
+              ...(filterTokenId !== undefined
+                ? { tokenId: filterTokenId }
+                : {}),
             },
           ]
         : await adapter.getProtocolTokens()
 
       const tokens = await Promise.all(
-        protocolTokens.map(async ({ address }) => {
+        protocolTokens.map(async ({ address, tokenId }) => {
           const startTime = Date.now()
 
           const unwrap = await adapter.unwrap({
             protocolTokenAddress: getAddress(address),
             blockNumber,
+            tokenId,
```

Comment:
We found that the defiProvider.unwrap function doesn't accept filterTokenId, even though the TestCase type has the filterTokenId property. 
https://github.com/consensys-vertical-apps/mmi-defi-adapters/blob/1c5bdabdceb5a77371d18e0beec7db49494ece87/src/types/testCase.ts#L73-L78

Also, the adapter.unwrap function has tokenId as  a parameter.
https://github.com/consensys-vertical-apps/mmi-defi-adapters/blob/1c5bdabdceb5a77371d18e0beec7db49494ece87/src/types/adapter.ts#L56-L70

We need tokenId to handle our stable pool LPs (erc-1155) correctly, so we decided to add tokenId as params.

It is outside of our adapter implementation, so please let me know if it is not acceptable.